scripts/nn/examples/fm-bin-class-dummy-data.dml

```diff
@@ -0,0 +1,48 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+#imports
+source("staging/fm-binclass.dml") as fmBinclass
+
+# generate dummy data ( this is just a sample! )
+n = 1000; d = 7; k=2;
+X = rand(rows=n, cols=d);
+y = rand(rows=n, cols=1);
```

Comment:
Since this is a classification problem, please round `y` to be in {0, 1}: `y = round(rand(rows=n, cols=1))`